content/millennium/dstu2/individuals/patient.md

```diff
@@ -145,6 +147,88 @@ The ability to perform patient combine or uncombine operations is not available
 
 The common [errors] may be returned.
 
+## Create
+
+Create an individual Patient:
+
+    POST /Patient
+
+_Implementation Notes_
+
+* The following elements are not supported and will not be used while creating the patient:
+    * [Patient.deceased]
+    * [Patient.multipleBirth]
+    * [Patient.photo]
+    * [Patient.contact]
+    * [Patient.animal]
+    * [Patient.managingOrganization]
+    * [Patient.link]
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/json+fhir', 'Content-Type': 'application/json+fhir'} %>
+
+### Body fields
+
+Notes:
+
+* Birth Sex may be recorded as an extension.
+
+<%= definition_table(:patient, :create, :dstu2) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Patient
+
+#### Body
+
+  <%= json(:dstu2_patient_create) %>
+
+#### Response
+
+  <%= headers status: 201 %>
+  <pre class="terminal">
+      Date → Tue, 27 Feb 2018 16:47:59 GMT
+      Cache-Control → no-cache
+      Vary → Origin,User-Agent,Accept-Encoding
+      Strict-Transport-Security → max-age=631152000
+      Server-Response-Time → 9272.410216999999
+      X-Xss-Protection → 1; mode=block
+      Pragma → no-cache
+      X-Request-Id → 78a19072002b8651623351cfedaffe70
+      Etag → W/"0"
+      X-Frame-Options → SAMEORIGIN
+      X-Runtime → 9.272318
+      X-Content-Type-Options → nosniff
+      Expires → Mon, 01 Jan 1990 00:00:00 GMT
+      Last-Modified → Tue, 27 Feb 2018 16:48:00 GMT
+      Location → https://fhir-ehr.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Patient/4842008
+      Status → 201 Created
+      Content-Length → 0
+      Content-Type → application/json
+  </pre>
+
+  The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
+
+
+### Errors
+
+The common [errors] may be returned. In addition, [OperationOutcomes] may be returned in the following scenarios:
```

Comment:
Need to fill in the OperationOutcomes link, it just shows up with no link and brackets right now

Reply:
Added the link.